src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css

```diff
@@ -0,0 +1,130 @@
+h5{
+    font-size: 20px;
+    color: rgb(0,83,78);
+  }  
+
+.appoinment-content{
+    overflow-y: scroll;
```

Comment:
In the UI there is no scroll bar

Reply:
I did not get you. I used the scroll bar in here
![Screenshot 2024-01-23 073741](https://github.com/ServeLankaHub/SLH-AngularJSWeb/assets/112056360/a544809c-fee7-4b64-8e45-69ae2ef81f7a)

Reply:
I mean in the figma design there is no scroll bar, are we going to add one?